### PR TITLE
fix: support multiple ou roots in OrgUnitSelector component

### PR DESCRIPTION
**Pull request by edoardo**
Fixes DHIS2-7679.

Changes proposed in this pull request:

- allow propagation of multiple ou roots from apps to `OrgUnitTreeMultipleRoots` used internally by `OrgUnitSelector`

### CODE REVIEW

File: packages/org-unit-dialog/src/OrgUnitSelector.js [hunk 1]
```diff
@@ -173,26 +192,39 @@ class OrgUnitSelector extends Component {
...
+                                    styles.orgUnitTree.selectedLabelStyle
+                                }
                                 checkboxColor={this.props.checkboxColor}
-                                displayNameProperty={this.props.displayNameProperty}
+                                displayNameProperty={
+                                    this.props.displayNameProperty
+                                }
                                 showFolderIcon
                                 disableSpacer
                             />
```
Comment (varl) [inline packages/org-unit-dialog/src/OrgUnitSelector.js]: This is the only functional change from what I can see. Is that correct?

Comment (edoardo) [reply packages/org-unit-dialog/src/OrgUnitSelector.js]: Sorry, after the last commit I lost reference here.
I believe you are referring to the addition of the {{roots}} prop, yes, that is the change.
I kept the prettier commit separated, so it's easier to see the actual change in commit: https://github.com/dhis2/d2-ui/pull/446/commits/936c2ba2bb9eff65824a247bf37224236f7d9bfa

File: packages/org-unit-tree/src/utils.js [hunk 2]
```diff
@@ -62,7 +82,12 @@ export const mergeChildren = (root, children) => {
  * @param forceReloadChildren
  * @returns {*}
  */
-export const loadChildren = (root, displayNameProperty, forceReloadChildren) => {
+export const loadChildren = (
+    root,
+    displayNameProperty,
+    forceReloadChildren,
+    useUserDataViewFallback
```
Comment (janhenrikoverland) [inline packages/org-unit-tree/src/utils.js]: Maybe call it "isUserDataViewFallback" so it doesn't look like a custom react hook.

Comment (edoardo) [reply packages/org-unit-tree/src/utils.js]: ok

Comment (edoardo) [reply packages/org-unit-tree/src/utils.js]: Changed

**Pull request merged**